go/enclave/enclave.go

```diff
@@ -326,12 +326,12 @@ func (e *enclaveImpl) GetRollupByHeight(rollupHeight uint64) *common.ExtRollup {
 	// TODO - Consider improving efficiency by directly fetching rollup by number.
 	rollup := e.storage.FetchHeadRollup()
 	for {
-		if rollup.Number().Cmp(rollupHeightBig) == 0 {
+		if rollup.Number().Uint64() == 0 {
```

Comment:
I don't think this is correct.
Shoul be: `if rollup.NumberU64() == rollupHeight`